applications/StructuralMechanicsApplication/custom_processes/assign_nodal_elements_to_nodes_process.h

```diff
@@ -0,0 +1,280 @@
+     */
+    void ExecuteInitialize() override;
+
+    /**
+     * @brief This function will be executed at every time step BEFORE performing the solve phase
+     */
+    void ExecuteInitializeSolutionStep() override;
+
+    /**
+     * @brief This method provides the defaults parameters to avoid conflicts between the different constructors
+     */
+    const Parameters GetDefaultParameters() const override
+    {
+        const Parameters default_parameters( R"({
+            "model_part_name"                : "",
+            "rayleigh_damping"               : false,
+            "interval"                       : [0.0, 1e30]
+        } )" );
+        return default_parameters;
+    }
+
+    ///@}
+    ///@name Access
+    ///@{
+
+    ///@}
+    ///@name Inquiry
+    ///@{
+
+    ///@}
+    ///@name Input and output
+    ///@{
+
+    /// Turn back information as a string.
+    std::string Info() const override
+    {
+        return "AssignNodalElementsToNodesProcess";
+    }
+
+    /// Print information about this object.
+    void PrintInfo(std::ostream& rOStream) const override
+    {
+        rOStream << "AssignNodalElementsToNodesProcess";
+    }
+
+    /// Print object's data.
+    void PrintData(std::ostream& rOStream) const override
+    {
+    }
+
+    ///@}
+    ///@name Friends
+    ///@{
+
+    ///@}
+protected:
+    ///@name Protected static Member Variables
+    ///@{
+
+    ///@}
+    ///@name Protected member Variables
+    ///@{
+
+    ///@}
+    ///@name Protected Operators
+    ///@{
+
+    ///@}
+    ///@name Protected Operations
+    ///@{
+
+    ///@}
+    ///@name Protected  Access
+    ///@{
+
+    ///@}
+    ///@name Protected Inquiry
+    ///@{
+
+    ///@}
+    ///@name Protected LifeCycle
+    ///@{
+
+    ///@}
+private:
+    ///@name Static Member Variables
+    ///@{
+
+
+    ///@}
+    ///@name Member Variables
+    ///@{
+
+    ModelPart& mrThisModelPart;   /// The main model part
+    Parameters mThisParameters;   /// The parameters (can be used for general pourposes)
+
+    ///@}
+    ///@name Private Operators
+    ///@{
+
+    ///@}
+    ///@name Private Operations
+    ///@{
+
+    /**
+     * @brief It returns a point geometry from an array of nodes
+     * @param rArrayNodes The array containing nodes
+     * @param Dimension The current dimension of work
+     * @return The pointer of the geometry of interest
+     */
+    GeometryType::Pointer GetPointGeometryFromNode(
+        PointerVector<Node>& rArrayNodes,
+        const SizeType Dimension
+        );
+
+    ///@}
+    ///@name Private  Access
+    ///@{
+
+    ///@}
+    ///@name Private Inquiry
+    ///@{
+
+    ///@}
+    ///@name Un accessible methods
+    ///@{
+
+    /// Assignment operator.
+    AssignNodalElementsToNodesProcess& operator=(AssignNodalElementsToNodesProcess const& rOther) = delete;
+
+    /// Copy constructor.
+    //AssignNodalElementsToNodesProcess(AssignNodalElementsToNodesProcess const& rOther);
```

Comment:
(Very minor) I'd put these in the life cycle and operators section of the header.

Reply:
Done